src/ui.js

```diff
@@ -26,7 +26,7 @@ export const makeTable = async () => {
       company[COMPANY_NAME_FIELD_NAME],
       company[STATUS_FIELD_NAME],
       company[CREATED_AT_FIELD_NAME],
-      company[REVENUE_YTD_FIELD_NAME],
+      company[REVENUE_YTD_FIELD_NAME].toLocaleString(),
```

Comment:
There are couple of problems with this:
1. Typically, the description of the task that needs to be done is given to you by product manager. By doing a different thing you basically says 'fuck you' to a product manager. Not good approach. If you feel you have a better idea, you should first talk to product manager who created the task, and only after getting his approval do things differently.
2. `toLocaleString()` will actually produce different formatting depending on a locale. In eu-US locale it might use commas as separator. But what outcome will be if your user has like Chinese locale or something?

Please make changes so that the result matches expected behavior described in the issue.

Reply:
Thanks for your comments! I realize now that this is an unacceptable solution. I will push a regex-based solution instead.